prisma/schema.prisma

```diff
@@ -106,7 +108,35 @@ model PlayerPosition {
 
   createdAt DateTime @default(now())
   updatedAt DateTime @updatedAt
-  
+
   @@map("playerPositions")
 }
 
+model User {
+  id Int @id @default(autoincrement())
+  email String?
+  firstname String?
+  lastname String?
```

Comment:
Can we add a username column and like realhub we can set default it to the email but eventually when we have a user management page we'll want to allow users to set a username maybe upload a profile pic etc..